scripts/nat/start-nat-node.sh

```diff
@@ -25,7 +25,9 @@ fi
 declare admin_port=${HOPRD_ADMIN_PORT:-3000}
 declare rest_port=${HOPRD_REST_PORT:-3001}
 declare healthcheck_port=${HOPRD_HEALTHCHECK_PORT:-8080}
+
 declare network_name="hopr-nat"
+declare container_name="hoprd-behind-nat"
```

Comment:
This will prevent the use of this script multiple times on the same machine. Can you parameterize this?

Reply:
Added env variable for this.